ui/lib/kv/addon/components/kv-version-dropdown.hbs

```diff
@@ -22,15 +22,6 @@
             </LinkTo>
           </li>
         {{/each}}
-        {{! version diff }}
-        {{#if (gt @metadata.sortedVersions.length 1)}}
-          <hr />
-          <li data-test-version="diff">
-            <LinkTo @route="secret.metadata.diff" {{on "click" (fn @onClose D)}}>
```

Comment:
this means we can delete this route in `ui/lib/kv/addon/routes.js` right?!

Reply:
I thought I did that, but yes